server/src/constants.go

```diff
@@ -39,6 +40,7 @@ const (
 	EndConditionStrikeout
 	EndConditionTimeout
 	EndConditionTerminated
+	EndConditionTerminatedByVote
```

Comment:
this has to be the final thing or else it will fuck up the database, meaning that it will shift all the ids and then the existing ids will no longer match

Reply:
makes sense

Reply:
There should be in golang something similar to c# to prevent constants renumbering e.g

```
enum ErrorCode : ushort
{
    None = 0,
    Unknown = 1,
    ConnectionLost = 100,
    OutlierReading = 200
}
```

Reply:
yes you can manually assign values in that way instead of using iota